### DISCUSSION TITLE
建议增加个AppImage.

### DISCUSSION THREAD

**Discussion by WackyKen** [pot-app/pot-desktop#377] [Ideas]
我使用 Deepin, 总是有依赖错误。原来还有个universal版，现在没了。从version 1.10.1开始就无法更新了。如果有个AppImage, 就不会有依赖性问题了。

**Comment by Pylogmon** [comment]
deepin的依赖问题早就解决了，deepin应用商店就可以安装。

**Reply by WackyKen** [reply]
V23安装不了啊。还是反应： tesseract_ocr_all 的broken dependency.

**Reply by Pylogmon** [reply]
V23是Deepin的问题，已经报告给Deepin去修了，tesseract依赖太大了，就算打包appimage我也不可能将他打包进去，你可以把deb直接解压来使用。

**Reply by Pylogmon** [reply]
只是系统OCR不能使用，其他都正常的。

**Reply by WackyKen** [reply]
OK, 非常感谢

**Reply by WackyKen** [reply]
OK, 非常感谢

**Comment by Lichenkass** [comment]
@WackyKen 此问题Deepin V23已修复
